### PR TITLE
rbd/tools: fix rwl related names

### PR DESCRIPTION
**Pull request by lixiaoy1**
These attributes have been renamed.

Signed-off-by: Li, Xiaoyan <xiaoyan.li@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/tools/rbd/action/Status.cc [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace {
 const std::string IMAGE_CACHE_STATE = ".librbd/image_cache_state";
 
 struct ImageCacheState {
-  bool present;
+  bool present = false;
```
Comment (lixiaoy1) [inline src/tools/rbd/action/Status.cc:27]: @dillaman  I did a test, it won't be initialized to false by default.

Comment (dillaman) [reply src/tools/rbd/action/Status.cc:27]: Might as well default initialize `clean` and `size` as well.

**Review by dillaman — APPROVED** [review]
lgtm

**Comment by dillaman** [comment]
(uninitialized variable also caused sporadic failures: http://qa-proxy.ceph.com/teuthology/jdillaman-2020-10-05_11:02:02-rbd-wip-jd-testing-distro-basic-smithi/5498536/teuthology.log)

**Comment by lixiaoy1** [comment]
@dillaman Thanks for the update.

**Pull request merged**